### DISCUSSION TITLE
ckan-2.9.7 Why Python3.8.10?

### DISCUSSION THREAD

**Discussion by To-Ki-O** [ckan/ckan#7361] [Q&A]
I'm sorry for bad english, but I'm Japanese.

I build ckan-2.9.7 with docker

I'm wondering why Python version is 3.8.10
The latest python3.8 is 3.8.16

Are there any problems?
If ckan is updated to 2.9.8, does it change for 3.8.16?

**Comment by kowh-ai** [comment]
@To-Ki-O - are you using these [instructions](https://docs.ckan.org/en/latest/maintaining/installing/install-from-docker-compose.html)  to build and run CKAN?

**Reply by To-Ki-O** [reply]
Thank you for replying.

yes. I read these instructions and did it, but I use Dockerfile.py3 to build CKAN.

**Reply by kowh-ai** [reply]
It doesn't really matter from a CKAN point of view if Python 3.8.10 is used. If you need to be specific you may wish to change the Dockerfile.py3  or you could have a go at installing manually into the running container

**Comment by To-Ki-O** [comment]
Thanks a lot!
I'll do it.
